task/06-conditions-n-loops-tasks.js

```diff
@@ -46,7 +55,12 @@ function getFizzBuzz(num) {
  *   10 => 3628800
  */
 function getFactorial(n) {
-    throw new Error('Not implemented');
+    let result = 1;
+    while(n){
```

Comment:
its not a good practice to change arguments if its not required. Also it's better to use for-loop when you know the number of step in advance